tests/test_txp_mau_dau.py

```diff
@@ -0,0 +1,21 @@
+from mozetl.testpilot.txp_mau_dau import get_experiments
+from utils import get_resource
+import requests
+import json
+
+
+class MockRequest():
+    def raise_for_status(self):
+        pass
+
+    def json(self):
+        return json.loads(get_resource('txp_experiments.json'))
+
+
+def test_get_experiments(monkeypatch):
+    def mock_get(url):
+        return MockRequest()
+    monkeypatch.setattr(requests, 'get', mock_get)
+    # There are 14 total experiments in the file but one should be filtered out
+    # due to not having an addon_id
```

Comment:
Updated -- but, given we're basing these mau/dau numbers off of the addons dataset, we don't really care about the non-addon experiments (and those are more accurately tracked on the serverside anyway since they're webapps)